docs/start_with_docker_plus_virtenv.md

```diff
@@ -0,0 +1,258 @@
+
+```bash
+[server-12] j='joint_wrist_yaw' idx=10 idx_q=10
+[server-12] j='joint_wrist_pitch' idx=11 idx_q=11
+[server-12] j='joint_wrist_roll' idx=12 idx_q=12
+[server-12] ==========================================
+[server-12] Starting up threads:                                    
+[server-12]  - Starting send thread
+[server-12]  - Starting recv thread
+[server-12]  - Sending state information
+[server-12]  - Sending servo information
+[server-12] Running all...                                          
+[server-12] Starting to send full state
+[stretch_driver-3] [INFO] [1727454898.725969113] [stretch_driver]: Changed to mode = position
+```
+
+## Create a virtual env on your GPU Computer
+
+First, check to see if mamba is installed on your computer:
+
+```bash
+mamba
+```
+
+If you get an "command not found" error, then follow the install instructions for mamba here: https://github.com/conda-forge/miniforge#download
+
+Then, run:
+
+```bash
+# Install Git LFS - needed for large files like images
+sudo apt-get install git-lfs
+git lfs install
+
+# Clone the repository
+# Do not forget the --recursive flag to clone submodules
+git clone git@github.com:hello-robot/stretch_ai.git --recursive
+
+# Install system dependencies
+# Pyaudio will fail if these are not installed
+sudo apt-get install libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0 espeak ffmpeg
+
+# Run install script to create a conda environment and install dependencies
+./install.sh
+```
+
+At the prompt to install Detic, say **no**. Take note of the name of the environment. It will be something like `stretch_ai_<version>`.
+
+Next, run:
+
+```bash
+# Activate your virtual env
+mamba activate stretch_ai_<version>
+
+# Tell mamba to not use your user sitepackages
+mamba env config vars set PYTHONNOUSERSITE=1
+
+# Install a version of setuptools for which clip works
+pip install setuptools==69.5.1
+
+# Install stretch-ai in "editable" mode so you can develop on it
+pip install -e ./src[dev]
+```
+
+Next, run:
+
+```bash
+# Install detectron2 for perception (required by Detic)
+git submodule update --init --recursive
+cd third_party/detectron2
+pip install -e .
+
+# Install Detic for perception
+cd ../../src/stretch/perception/detection/detic/Detic
+# Make sure it's up to date
+git submodule update --init --recursive
+pip install -r requirements.txt
+
+# Download DETIC checkpoint...
+mkdir -p models
+wget --no-check-certificate https://dl.fbaipublicfiles.com/detic/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.pth -O models/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.pth
+```
+
+**Note:** A new env should be created for each new version of *stretch-ai* (each time you `git pull`) using the above instructions.
+
+#### Activate your virtual env
+
+```bash
+mamba activate stretch_ai_<version>
+```
+
+#### Verify NVIDIA Docker
```

Comment:
Why is this included in the start w.o docker section

Reply:
Good catch. Removing.